src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -171,7 +171,7 @@ internal View TemplatedRoot
 						RegisterContentTemplateRoot();
 
 						if (
-#if NETSTANDARD
+#if __NETSTD__
```

Comment:
I think following is the origin of the bug.

1. In a previous PR of @dr1rrb, he had `IsLoading` unconditionally.
2. Then, build fails for some targets.
3. Check where `IsLoading` is defined and find it is only in `UIElement.netstd.cs`
4. Add `#if NETSTANDARD`, which makes sense, that is supposed to be the only target with `IsLoading` defined. But.. that's not correct.

`*.netstd.cs` doesn't actually refer to .NET Standard. Our definition is:

```csharp
		<_IsNetStd Condition="'$(TargetFramework)'=='netstandard2.0' or $(_IsNetCore)">true</_IsNetStd>
```

The root of issue is really bad naming.

We should probably find a better name for `_IsNetStd` property and also rename our `.netstd.cs` files accordingly. But that is totally a separate task.

Reply:
@MartinZikmund @Youssef1313 It would be interesting to have a runtime test that validates this particular behavior. It looks like we do not validate the size of the popup.